database/main.tf

```diff
@@ -1,16 +1,17 @@
-data "cloudfoundry_space" "space" {
-  org_name = var.cf_org_name
-  name     = var.cf_space_name
+locals {
+  tags = setunion(["terraform-cloudgov"], var.tags)
 }
 
-data "cloudfoundry_service" "rds" {
-  name = "aws-rds"
+data "cloudfoundry_service_plans" "rds" {
+  name                  = var.rds_plan_name
+  service_offering_name = "aws-rds"
 }
 
 resource "cloudfoundry_service_instance" "rds" {
   name         = var.name
-  space        = data.cloudfoundry_space.space.id
-  service_plan = data.cloudfoundry_service.rds.service_plans[var.rds_plan_name]
-  tags         = var.tags
-  json_params  = var.json_params
+  space        = var.cf_space_id
+  type         = "managed"
+  service_plan = data.cloudfoundry_service_plans.rds.service_plans.0.id
```

Comment:
Shouldn't `var.rds_plan_name` be used to index into `service_plans` here?

Reply:
Never mind, I just learned that supplying `name = var.rds_plan_name` to `cloudfoundry_service_plans` will filter the result to just that one service plan, so there will only be one result and `.0.` is always right!